src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -194,6 +194,64 @@ ngraph::pass::GeluFusionWithErfThree::GeluFusionWithErfThree() {
     register_matcher(m, callback);
 }
 
+ngraph::pass::GeluFusionWithErfFour::GeluFusionWithErfFour() {
+    MATCHER_SCOPE(GeluFusionWithErfFour);
+
+    auto input = ngraph::pattern::any_input();
+    auto mul1_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    auto mul1 = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({input, mul1_constant});
+    auto erf = ngraph::pattern::wrap_type<ngraph::opset7::Erf>({mul1});
+    auto mul2_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    auto mul2 = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({erf, mul2_constant});
+    auto add_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    auto add = ngraph::pattern::wrap_type<ngraph::opset7::Add>({add_constant, mul2});
+
+    // x * (0.5 + 0.5 * erf(x * (1 / sqrt(2)))
+    auto mul3 = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({input, add});
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto& pattern_to_output = m.get_pattern_value_map();
```

Comment:
it's better to use `get_pattern_map`, not `get_pattern_value_map` to avoid useless .get_node_shared_ptr()